swagger-spec/licenses/definition.yaml

```diff
@@ -0,0 +1,58 @@
+# Model definition for licenses
+type: object
+title: License
+required:
+  - id
+  - type
+properties:
+  links:
+    type: object
+    readOnly: true
+    description: 'URLs to alternative representations of the license.'
+    properties:
+      self:
+        type: string
+        format: URL
+        readOnly: true
+        description: 'A link to the detail page for the license.'
+  id:
+    type: string
+    readOnly: true
+    description: 'The identifier of the license.'
+  type:
+    type: string
+    readOnly: true
+    description: 'The type identifier of the license (`license`).'
+  attributes:
+    type: object
+    readOnly: true
+    description: 'The properties of the license.'
+    properties:
+      name:
+        type: string
+        readOnly: true
+        description: 'Name of the license.'
+      text:
+        type: string
+        readOnly: true
+        description: 'Full text of the license.'
+      required_fields:
+        type: array
+        readOnly: true
+        description: 'Fields required for this license (provided to help front-end validators). Maps to properties on the NodeLicense model.'
+example:
```

Comment:
Since licenses are read only, I don't think the `example` key is necessary in this definition. The `example` key in a definition is what forms an example request payload, but let me know if it's being used differently here.